Create ui-c8y-1023-28-6-data-point-table-template-link-fix-10884-graft-release-cd-11100.md (#4213)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-28-6-data-point-table-template-link-fix-10884-graft-release-cd-11100.md

---
date: ""

> title: data point table template linking fix

Reviewer: ```suggestion
title: Fixed issue with data point template linking
```

> title: Fixed issue with data point template linking

Reviewer: ```suggestion
title: Fixed issue with data point template
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65426
version: 1023.28.6
---

> An issue was identified where data point templates could overwrite the device ID, leading to unexpected behavior in some widgets. The linkage between templates and data points has been improved, resolving these issues and ensuring more reliable widget behavior.

Reviewer: ```suggestion
An issue was identified where data point templates could overwrite the device ID, leading to unexpected behavior in some widgets. The linkage between templates and data points has been improved, resolving this issue and ensuring a more reliable widget behavior.
```

---
date: ""
title: Data point table template link fix. (#10884) [GRAFT][release/cd] (#11100)
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65426
version: 1023.28.6
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-65426] Data point table template link fix.
(#10884)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/10884)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-65426]:
https://cumulocity.atlassian.net/browse/MTM-65426?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>

Reviewer 2: ```suggestion
An issue was identified where data point templates could overwrite the device ID, leading to unexpected behavior in some widgets. The linkage between templates and data points has been improved, resolving these issues and ensuring more reliable widget behavior.
```
